packages/runtime/src/widgets/Widgets.css

```diff
@@ -0,0 +1,3 @@
+.no-box-shadow:focus {
+  box-shadow: none !important;
+}
```

Comment:
Can we stick to using CSS-in-JS for now? We need to set up bundling stylesheets separately